src/test/java/seedu/address/model/person/DeliveryDateTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+
+
+public class DeliveryDateTest {
```

Comment:
Looks pretty thorough to me, nice

Reply:
I'm not sure if this may be necessary, but I think it could be good to have two date strings, parse them, and make sure they're equal in the date value represent? Just to check that the equals method doesn't break

Reply:
Good catch! Will do.